pkg/sloop/server/internal/config/config.go

```diff
@@ -172,16 +173,31 @@ func (c *SloopConfig) Validate() error {
 }
 
 func loadFromFile(filename string) *SloopConfig {
-	yamlFile, err := ioutil.ReadFile(filename)
-	if err != nil {
-		panic(fmt.Sprintf("failed to read %v. %v", filename, err))
-	}
-	var config SloopConfig
-	err = yaml.Unmarshal(yamlFile, &config)
-	if err != nil {
-		panic(fmt.Sprintf("failed to unmarshal %v. %v", filename, err))
+	if strings.Contains(filename,".yaml") {
+		yamlFile, err := ioutil.ReadFile(filename)
```

Comment:
Can you please add unit tests for this case?

Reply:
I'll have to include two files with extensions .yaml and .json to the folder. what i can do is use filepath.ext(file_url)  to explain what this does?

Reply:
yes you would need to add mock files to test this code

Reply:
done!